packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -115,6 +116,19 @@ export default class CleanWebpackPlugin {
     this.removeFiles = this.removeFiles.bind(this);
   }
 
+  private checkIfFileNameIsDuplicate(): boolean {
```

Comment:
I'd prefer if you simply used an external fn outside the class. Using a private method does not really present any advantage and may may it even harder to test later

Reply:
I see, I may have to entirely remove it, depending on the conclusion of the next conversation. However, if I do end up keeping it, I will move it to a separate function outside this class.